### PR TITLE
Updated e2e tests failing due to JobProperties casting issue

### PR DESCRIPTION
**Pull request by tmahmood-microsoft**

### CODE REVIEW

File: iothub/service/src/Registry/DevicesClient.cs [hunk 1]
```diff
@@ -567,7 +567,7 @@ public virtual async Task<IotHubJobResponse> ImportAsync(ImportJobProperties job
 
             try
             {
-                return await CreateJobAsync<ImportJobProperties>(jobParameters, cancellationToken).ConfigureAwait(false);
```
Comment (timtay-microsoft) [inline iothub/service/src/Registry/DevicesClient.cs:570]: I'm a little confused, now. I thought the point of the ImportJobProperties class was to be returned in this method. Are we actually using the importJobProperties class anymore?

Comment (tmahmood-microsoft) [reply iothub/service/src/Registry/DevicesClient.cs]: Fixed the change. 
The return type of importAsync / exportAsync should be ImportJobProperties / ExportJobProperties.
GetJobAsync() should return instance of IotHubJobResponse

**Review by timtay-microsoft — APPROVED** [review]

**Review by drwill-ms — APPROVED** [review]

**Pull request merged**